presto-orc/src/test/java/com/facebook/presto/orc/metadata/TestDwrfMetadataWriter.java

```diff
@@ -0,0 +1,92 @@
+import static com.facebook.presto.orc.metadata.ColumnEncoding.ColumnEncodingKind.DIRECT;
+import static com.facebook.presto.orc.metadata.ColumnEncoding.ColumnEncodingKind.DWRF_MAP_FLAT;
+import static com.facebook.presto.orc.metadata.DwrfMetadataWriter.toColumnEncoding;
+import static com.facebook.presto.orc.metadata.DwrfMetadataWriter.toStreamKind;
+import static com.facebook.presto.orc.metadata.Stream.StreamKind.DATA;
+import static com.facebook.presto.orc.metadata.Stream.StreamKind.DICTIONARY_COUNT;
+import static com.facebook.presto.orc.metadata.Stream.StreamKind.DICTIONARY_DATA;
+import static com.facebook.presto.orc.metadata.Stream.StreamKind.IN_MAP;
+import static com.facebook.presto.orc.metadata.Stream.StreamKind.LENGTH;
+import static com.facebook.presto.orc.metadata.Stream.StreamKind.PRESENT;
+import static com.facebook.presto.orc.metadata.Stream.StreamKind.ROW_INDEX;
+import static com.facebook.presto.orc.metadata.Stream.StreamKind.SECONDARY;
+import static org.testng.Assert.assertEquals;
+
+public class TestDwrfMetadataWriter
+{
+    private static final int COLUMN_ID = 3;
+
+    @Test
+    public void testToColumnEncodingDirect()
+    {
+        int expectedDictionarySize = 0;
+        ColumnEncoding columnEncoding = new ColumnEncoding(DIRECT, expectedDictionarySize);
+
+        DwrfProto.ColumnEncoding actual = toColumnEncoding(COLUMN_ID, columnEncoding);
+
+        assertEquals(actual.getColumn(), COLUMN_ID);
+        assertEquals(actual.getKind(), DwrfProto.ColumnEncoding.Kind.DIRECT);
+        assertEquals(actual.getDictionarySize(), expectedDictionarySize);
+        assertEquals(actual.getSequence(), 0);
+    }
+
+    @Test
+    public void testToColumnEncodingDictionary()
+    {
+        int expectedDictionarySize = 5;
+        ColumnEncoding columnEncoding = new ColumnEncoding(DICTIONARY, expectedDictionarySize);
+
+        DwrfProto.ColumnEncoding actual = toColumnEncoding(COLUMN_ID, columnEncoding);
+
+        assertEquals(actual.getColumn(), COLUMN_ID);
+        assertEquals(actual.getKind(), DwrfProto.ColumnEncoding.Kind.DICTIONARY);
+        assertEquals(actual.getDictionarySize(), expectedDictionarySize);
+        assertEquals(actual.getSequence(), 0);
+    }
+
+    @Test
+    public void testToColumnEncodingFlatMap()
+    {
+        int expectedDictionarySize = 0;
+        ColumnEncoding columnEncoding = new ColumnEncoding(DWRF_MAP_FLAT, expectedDictionarySize);
+
+        DwrfProto.ColumnEncoding actual = toColumnEncoding(COLUMN_ID, columnEncoding);
+
+        assertEquals(actual.getColumn(), COLUMN_ID);
+        assertEquals(actual.getKind(), DwrfProto.ColumnEncoding.Kind.MAP_FLAT);
+        assertEquals(actual.getDictionarySize(), expectedDictionarySize);
+        assertEquals(actual.getSequence(), 0);
+    }
+
+    @Test
+    public void testToStreamKind()
+    {
+        assertEquals(toStreamKind(PRESENT), DwrfProto.Stream.Kind.PRESENT);
```

Comment:
I used the same import schema for these deeply nested class as in the DwrfMetadataWriter.